build/docker.mk

```diff
@@ -23,7 +23,8 @@ IMAGE_SRC?=https://github.com/radius-project/radius
 # Generate a target for each image we define
 # Params:
 # $(1): the image name for the target
-# $(2): the Dockerfile path
+# $(2): additional arguments (if it is a dot, it means the current directory)
```

Comment:
Is this actually `additional arguments`? or is it the "context directory"? https://docs.docker.com/build/building/context/